packages/react-ui/src/theme/sections/components/dataDisplay.js

```diff
@@ -223,6 +223,7 @@ export const dataDisplayOverrides = {
         border: `2px solid ${commonPalette.divider}`,
         borderRadius: getSpacing(0.5),
         fontWeight: themeTypography.fontWeightMedium,
+        paddingRight: getSpacing(3),
```

Comment:
IMO there are a few other changes that would need to be done, reducing font sizes in the paginator, but I'm adding here what I see is the biggest pain

Please @vmilan create a task for reviewing with Figma

Reply:
I'm landing for the moment, with current approach

Reply:
We already have a ticket for the pagination issue: https://app.shortcut.com/cartoteam/story/308963/fix-the-select-component-from-the-table-pagination

I'll take a look to see if there is something more to be fixed.

Thanks for taking care of these PR's @VictorVelarde !